autotest/README.md

```diff
@@ -0,0 +1,104 @@
+# autotest case
+
+We provide a autotest caseset to do regression.
+
+## How to run testcases
+
+Install required dependencies using the following command line:
+
+```bash
+python3 -m pip install -r requirements/test.txt
+```
+
+Run pytest command line with case filtering through -m flag. eg: `-m internlm_chat_7b` Filter cases related to internlm_chat_7b. The corresponding results will be stored in the `allure-results` directory.
+
+```bash
+pytest autotest -m internlm_chat_7b --clean-alluredir --alluredir=allure-results
+```
+
+After the test cases are executed, you can generate the test report by using the following command. The generated report will be stored in `allure-reports`.
+
+```bash
```

Comment:
add installation of allure: https://allurereport.org/docs/gettingstarted-installation/#install-via-the-system-package-manager-for-linux
```
wget https://github.com/allure-framework/allure2/releases/download/2.25.0/allure_2.24.1-1_all.deb
sudo apt-get install -y openjdk-8-jre-headless
sudo dpkg -i ./allure_2.24.1-1_all.deb
```